hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/Table.java

```diff
@@ -337,12 +337,8 @@ interface KeyValue<KEY, VALUE> {
 
     VALUE getValue() throws IOException;
 
-    default byte[] getRawKey() throws IOException {
-      return null;
-    }
-
-    default byte[] getRawValue() throws IOException {
-      return null;
+    default int getReadableBytes()  throws IOException {
+      return -1;
```

Comment:
I think 0 would be safer for size calculations.

`consumedSize += objectSerializedSize` etc.